src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -60,7 +61,7 @@ export const MapRoom: React.FC<MapRoomProps> = ({
       <img className="maproom__image" src={room.image_url} alt={room.title} />
 
       <div className="maproom__label">
-        {room.title}
+        <span className="maproom__roomtitle">{room.title}</span>
```

Comment:
I think we can be even simpler here, and just conditionally render the `room.title` in the React/JSX directly. No need to add the extra CSS/etc for it.

You can read more about this here:

- https://reactjs.org/docs/conditional-rendering.html

Specifically this part:

- https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

---

By doing it this way, you can then probably clean up/remove all of the extra CSS changes that you made in this PR.